### PR TITLE
Add basic resources.md

### PR DESCRIPTION
**Pull request by MaxGraey**
Start implementing structure described in #1

### CODE REVIEW

File: resources/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Official Website
+ - https://webassembly.org
+
+## Specifications
+ - [Core on webpage](https://webassembly.github.io/spec/core)
+ - [Core on PDF](https://webassembly.github.io/spec/core/_download/WebAssembly.pdf)
+ - [Core on Github](https://github.com/WebAssembly/design)
```
Comment (chicoxyzzy) [inline resources/README.md]: those are design docs, not spec

**Comment by chicoxyzzy** [comment]
I think that https://webassembly.org/specs/ instead of all links under "Specifications" header will be enough

**Comment by MaxGraey** [comment]
You could improve all of this afterwards. Just want to start some basics

File: resources/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+### [Official Website](https://webassembly.org)
+
+## Documentation
+
+#### Specifications
+ - [Core on webpage](https://webassembly.github.io/spec/core)
+ - [Core on PDF](https://webassembly.github.io/spec/core/_download/WebAssembly.pdf)
```
Comment (chicoxyzzy) [inline resources/README.md]: I think that there is no need in having PDF link here, but JS API, Web API, and WASI specs would be helpful. https://webassembly.org/specs/ could be enough here

File: resources/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+ - [Awesome Wasm](https://github.com/mbasso/awesome-wasm)<br/>
+ Collection of general awesome things regarding WebAssembly ecosystem.
+ - [Awesome WebAssembly Languages](https://github.com/appcypher/awesome-wasm-langs)<br/>
+ Collection of languages that currently compile to or have their VMs in WebAssembly.
+ - [Awesome WebAssembly Runtimes](https://github.com/appcypher/awesome-wasm-runtimes)<br/>
+ Collection of virtual machines and tools that execute the WebAssembly.
+
+## Community & Social Media
+ - [Community Group](https://www.w3.org/community/webassembly)
```
Comment (chicoxyzzy) [inline resources/README.md:46]: Probably this link is not so helpful. I think Discord chat and Telegram chat links will be more helpful from what people could expect from the header

**Review by chicoxyzzy — APPROVED** [review]
I left some proposal. Overall looks great.

File: resources/README.md [hunk 4]
```diff
@@ -3,8 +3,7 @@
 ## Documentation
 
 #### Specifications
- - [Core on webpage](https://webassembly.github.io/spec/core)
- - [Core on PDF](https://webassembly.github.io/spec/core/_download/WebAssembly.pdf)
+ - [Core](https://webassembly.github.io/spec)
```
Comment (chicoxyzzy) [inline resources/README.md]:
```suggestion
 - [List of the specs on official website](https://webassembly.github.io/spec)
```

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**